Submodels/file_handling.py

```
@@ -0,0 +1,689 @@
+        
+        This method sets up the initial state of the class.
+        The fileHandling class is, in effect, a grouping class, 
+        and therefore does not take any inputs.
+
+        Returns
+        -------
+        None
+        """
+
+
+    class fileHandlingMTSET():
```

Ruff (0.8.2)

Unnecessary parentheses after class definition

Remove parentheses

(UP039)